engine/modules/opentelemetry/inc/com/centreon/engine/modules/opentelemetry/opentelemetry.hh

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2024 Centreon
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ */
+#ifndef CCE_MOD_OTL_SERVER_OPENTELEMETRY_HH
+#define CCE_MOD_OTL_SERVER_OPENTELEMETRY_HH
+
+#include "com/centreon/engine/commands/otel_interface.hh"
+
+#include "otl_data_point.hh"
+#include "otl_config.hh"
+
+namespace com::centreon::engine::modules::opentelemetry {
+
+using host_serv_metric = commands::otel::host_serv_metric;
+
+class otl_server;
+
+/**
+ * @brief This class is the high level main class of otel module
+ * It needs a json config file witch path is passed to load and an io_context
+ * for second timer
+ * The two methods used during runtime is create_extractor and check
```

Comment:
```suggestion
 * @brief This class is the main high-level class of the otel module.
 * It needs a json configuration file whose path is passed on loading, and an io_context
 * for the second timer.
 * The two methods used at runtime are create_extractor() and check().
```

Reply:
You don't explain what is second_timer?
If it is the second one, where is the first one?